packages/select/src/Select/components/ListBox/ListBox.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from "react"
+import { Node } from "@react-types/shared"
+import classNames from "classnames"
+import { useSelectionContext } from "../../provider/SelectionProvider"
+import { ItemType } from "../../types"
+import styles from "./ListBox.module.scss"
+
+export interface ListBoxProps {
+  children: (items: {
+    selectedItems: Array<Node<ItemType>>
+    unselectedItems: Array<Node<ItemType>>
+    allItems: Array<Node<ItemType>>
+  }) => React.ReactNode
+}
+
+export const ListBox: React.VFC<ListBoxProps> = ({ children }) => {
+  const { listBoxProps, listRef, selectionState } = useSelectionContext()
+  const [isOverflown, setIsOverflown] = useState(false)
+  useEffect(() => {
+    const listElement = listRef.current
+    if (!listElement) {
+      return
+    }
+    setIsOverflown(listElement.scrollHeight > listElement.clientHeight)
```

Comment:
This seems excessive to remove the margin, why does the margin exist in the first place?

Reply:
I just checked, the overflow class which add `margin-right: 0;` has no effect on the component, all that logic can be removed